ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1294,18 +1306,6 @@ tests:
     - ref: cucushift-helper-service-build-info
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-hypershift
-- as: azure-ipi-ovn-hypershift-p3-f21
```

Comment:
The relocation makes the tests unsorted.
```
azure-ipi-ovn-hypershift-install-p3-f21
azure-ipi-ovn-hypershift-p3-f21
```

Reply:
Sorry, let me update it. Thanks a lot!